csm/core/services/storage_capacity.py

```diff
@@ -133,6 +134,10 @@ async def get_cluster_data(self, data_filter=None):
                     else:
                         time.sleep(RETRY_SLEEP_DURATION)
                         continue
+                except asyncio.TimeoutError as e:
+                    Log.error(f"Error in obtaining response from {url}:{e}")
```

Comment:
This log statement should clearly state that exception is due to "TimeOut". Please update accordingly.

Reply:
It will print log as:
Error in obtaining response from <url>: TimeoutError